modules/btn-actions.js

```diff
@@ -0,0 +1,39 @@
+// ------------------ Opdracht 1a -------------------
+
+// const tvNamesType = inventory.map((tvtype) => tvtype.type);
+// console.table(tvNamesType);
+
+// ------------------ Opdracht 1b - tv's all sold-out -------------------
+
+function handleTvSoldOut(arr) {
+  
+ return arr.filter((soldOut) => soldOut.originalStock - soldOut.sold === 0);
```

Comment:
Kan nog ietjes korter 😉 
```suggestion
 return arr.filter((soldOut) => soldOut.originalStock === soldOut.sold);
```

Reply:
Dat ik dit over het hoofd hebt gezien. Helemaal gelijk dit scheelt weer in code :-)